src/main/java/nextstep/subway/domain/pathfinder/PathFinder.java

```diff
@@ -0,0 +1,50 @@
+package nextstep.subway.domain.pathfinder;
+
+import nextstep.subway.application.dto.PathResponse;
+import nextstep.subway.domain.CustomWeightedEdge;
+import nextstep.subway.domain.PathType;
+import nextstep.subway.domain.Section;
+import nextstep.subway.domain.Station;
+import org.jgrapht.GraphPath;
+import org.jgrapht.alg.shortestpath.DijkstraShortestPath;
+import org.jgrapht.graph.WeightedMultigraph;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+
+public abstract class PathFinder {
+    abstract public boolean isType(PathType pathType);
+    public PathResponse findPath(final List<Section> sections, final Station sourceStation, final Station targetStation) {
```

Comment:
패키지 분리와 추상 클래스로 변경해주셨네요 👍 
적절한 개행으로 가독성도 챙기면 좋을 것 같아요 😃 
```suggestion
public abstract class PathFinder {

    abstract public boolean isType(PathType pathType);
    
    public PathResponse findPath(final List<Section> sections, final Station sourceStation, final Station targetStation) {
```